base/docs/helpdb/Base.jl

```diff
@@ -4057,7 +4057,8 @@ abs2
     write(filename::AbstractString, x)
 
 Write the canonical binary representation of a value to the given I/O stream or file.
-Returns the number of bytes written into the stream.
+Returns the number of bytes written into the stream. *`write(::IOStream, ...)` is a synchronous operation and will block at the process level.*
```

Comment:
These should be line wrapped, and probably migrated inline while you're adjusting them.